docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -325,3 +325,42 @@ This issue might occur when you use a Zowe version of 1.12.0 or later. To resolv
 ```ZWED_node_https_certificateAuthorities="/path/to/zowe/keystore/local_ca/localca.cer-ebcdic","/path/to/carootcert.pem","/path/to/caintermediatecert.pem"```
  
 Recycle your Zowe server. You should be able to log in to the Zowe Desktop successfully now.
+
+### Browser unable to connect with CIPHER_MISMATCH errors
+
+**Symptom:**
+
+When connecting to the API Mediation Layer the web browser throws an error that the site is unable to provide a secure connection because of a `CIPHER_MISMATCH`.  
+
+The error shown varies depending on the browser.
+
+<img src="../images/common/cipher_mismatch.png" alt="CIPHER_MISMATCH" title="CIPHER_MISMATCH Error"/>
+
+For Firefox it might be:
```

Comment:
This line implies that there will be an example of the Firefox error after it, but I don't see the example